### PR TITLE
Localize BanCommand

### PR DESCRIPTION
**Pull request by Pr0methean**
This localizes BanCommand and also demonstrates how localized commands can work. Since each CommandSender has a different ResourceBundle, all LocalizedStringImpl instances a command uses must be created on demand, and thus LocalizedStringsTest's approach isn't applicable here.

### CODE REVIEW

File: src/main/java/net/glowstone/i18n/LocalizedString.java [hunk 1]
```diff
@@ -8,4 +9,6 @@
     String get(Object... args);
 
     void send(CommandSender recipient, Object... args);
+
+    void sendInColor(CommandSender recipient, ChatColor color, Object... args);
```
Comment (mastercoms) [inline src/main/java/net/glowstone/i18n/LocalizedString.java:13]: I would prefer if we also supported multiple color codes within the message with the section sign (`§`).

But perhaps it's out of the scope of this PR.

**Review by mastercoms — APPROVED** [review]
Very well done!

File: src/main/java/net/glowstone/i18n/LocalizedString.java (hunk 1 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/i18n/LocalizedString.java:13]: For multiple colors, you'd want to add the color-codes to the resource strings themselves anyway, to indicate which part was which color.

Comment (mastercoms) [reply src/main/java/net/glowstone/i18n/LocalizedString.java:13]: Yes, this is what I meant. Does that work already?

**Pull request merged**